modelmapper/cleaner.py

```diff
@@ -60,16 +72,35 @@ def get_csv_data_cleaned(self, path_or_content, original_content_type=None):
             try:
                 field_info = model_info[field_name]
             except KeyError:
-                raise KeyError(FIELD_NAME_NOT_FOUND_MSG.format(field_name)) from None
+                if ignore_new_fields:
+                    self._missing_fields.add(field_name)
+                    continue
+                else:
+                    raise KeyError(FIELD_NAME_NOT_FOUND_MSG.format(field_name)) from None
             self._get_field_values_cleaned_for_importing(field_name, field_info, field_values, original_content_type)
+        if ignore_new_fields and self._missing_fields:
+            for field in self._missing_fields:
+                all_items.pop(field)
+
+            if not self.publicized:
+                self.logger.error(
```

Comment:
I'd prefer this message be the same one as in the `slack_missing_fields`. Also, same notes about usage of the triple quotes

Reply:
👍

Reply:
👍